packages/__cxstarter__/src/components/FlowContainer/tokens/__cxstarter__FlowContainer.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright © 2022 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { asFluidToken } from '@bodiless/cx-elements';
+import { cxFlowContainer } from '@bodiless/cx-flowcontainer';
+// import { withRichTextVariations } from '../RichText';
+
+const Default = asFluidToken({
+  ...cxFlowContainer.Default,
+  Components: {
+    ...cxFlowContainer.Default.Components,
+    // ...withRichTextVariations.Components,
```

Comment:
I don't think we need this as we are shadowing the RichText tokens.  We would only need to add this if we were adding a *new* rich text varition.